modules/nw-dns-forward.adoc

```diff
@@ -5,7 +5,7 @@
 [id="nw-dns-forward_{context}"]
 = Using DNS forwarding
 
-You can use DNS forwarding to override the forwarding configuration identified in `/etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone.
+You can use DNS forwarding to override the forwarding configuration identified in `/etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
```

Comment:
Suggested edit: 
If the forwarded zone is a domain managed by {product-title}, such as the Ingress domain, then...